src/components/Image/Image.md

```diff
@@ -124,19 +120,21 @@ const yammerLogoDescription = 'Yammer "y" logo';
 Full width:
 
 ```js { "props": { "data-example": "full width" } }
+const { GutterSize } = require('../FixedGrid');
 const { ImageFit } = require('.');
-const { FixedGridColumn, FixedGridRow, GutterSize } = require('../FixedGrid')
 
 const yammerLogo = 'yammer-logo.png';
 const yammerLogoDescription = 'Yammer "y" logo';
 
 const fullWidthImage = (
-  <Image source={yammerLogo} description={yammerLogoDescription} fullWidth />
+  <Image source={yammerLogo} description={yammerLogoDescription} fullWidth={true} />
```

Comment:
whoops, i had removed all these not knowing we have new linting rules to require boolean attribute values (this wasn't caught since it's code in markdown). sorry about that, looks like you fixed them all :/

Reply:
No worries.